docs/src/main/paradox/docs/faq.md

```diff
@@ -121,8 +121,7 @@ exploration of trends and patterns.
 
 @link:[SHACL](https://www.w3.org/TR/shacl/){ open=new } (Shapes Constraint Language) is a language for validating RDF 
 graphs against a set of conditions. These conditions are provided as shapes and other constructs expressed in the form 
-of an RDF graph. SHACL is used in Blue Brain Nexus to constrain and control the payload that can be pushed into Nexus. 
-You can use the @link:[SHACL Playground](https://shacl.org/playground/){ open=new } to test your schemas.
+of an RDF graph. SHACL is used in Blue Brain Nexus to constrain and control the payload that can be pushed into Nexus.
```

Comment:
Revert this change, since they have already fixed it https://github.com/TopQuadrant/shacl-js/issues/29

Reply:
thanks for checking! ;)